DeratControl.Application/Requests/CommandExecutionContext.cs

```diff
@@ -7,6 +7,12 @@ namespace DeratControl.Application.Requests.Interfaces
 {
     public class CommandExecutionContext
     {
-        public User RequestedUser { get; set; }
+        private User requestedUser;
+        public User RequestedUser { get;}
```

Comment:
do you need a private field for that? Read only properties can be set in a constructor